LABTASKS.md

```diff
@@ -1,25 +1,100 @@
-    location. Your style sheet should make the web page look nice for both a phone and
-    a computer screen, and your choices should not interfere with accessibility in
-    obvious ways.
-  - Be sure to follow the setup instructions in the [README.md](README.md) to set up
-    GitHub pages (including changing the line in the `Running your project` section of
-    the [README.md](README.md) to link to your team's GitHub pages page).
-
-## Technical Requirements
+Your tasks for this lab are:
+
+- To add a small bit of HTML using classes and IDs to allow you to format
+  different components in different ways.
+- To develop an additional, new look for a web page using different CSS without altering
+  the HTML.
+
+All of the formatting (including the positioning of elements in the page) must
+  be done using CSS. **Do not alter the HTML** with two exceptions:
+
+- You have to extend the HTML in the first part. You should do that before you do
+  any of the CSS work.
+- If you want to load Google Fonts you'll want to add those tags to your HTML so
+  they'll be available in your CSS. (There are ways to do this entirely in CSS
+  but the "standard" approach typically involves modifying the HTML.)
+
+Be intentional and sparing with your use of absolute sizes for things like font and
+location. Your style sheet should make the web page look nice for both a phone and
+a computer screen, and your choices should not interfere with accessibility in
+obvious ways.
+
+## Extending the HTML
+
+The current HTML has three major sections:
+
+- CSci Program Student Learning Outcomes (PSLOs)
+- CSci major requirements
+- CSci faculty
+
+You should add a new section to the HTML (without changing any of the existing HTML)
+that provides a list of some of the tools we'll be using in this class:
+
+- [`git`](https://git-scm.com/)
+- [GitHub](https://github.com/)
+- [GitKraken](https://www.gitkraken.com/git-client)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [JUnit](https://junit.org/)
+- [Karma](https://karma-runner.github.io/latest/index.html)
+- [Protractor](https://www.protractortest.org/#/)
+- [GitHub Actions](https://github.com/features/actions)
+
+Some requirements:
+
+- Add some sort of header for this section that indicates what information is
+  being presented.
+- Each item should be be a link to the same URL as provided in the links in the
+  list above.
+- The following items should be in class "cloud": GitHub.com and GitHub Actions.
+  Every other item should be in the class "local".
+- All items should be in a class indicating that tool's area of functionality:
+  - `version-control` for `git`, GitHub.com, and GitKraken`
+  - `ide` for Visual Studio Code
+  - `testing` for JUnit, Karma, Protractor, and GitHub Actions
+
+You can provide multiple, space-separated classes for an item, e.g.,
+
+```html
+    <li class="local version-control" id="git"><code>git</code></li>
```

Comment:
This implies that all the elements in the list should be in code blocks, maybe provide two examples here like:
```html
<li class="local version-control" id="git"><a href="https://git-scm.com/"><code>git</code></a></li>
<li class="local ide" id="vscode"><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
```

Also I noticed that the examples aren't links, they probably should be since that is requested.

Reply:
Good catch – thanks!

Reply:
Dealt with this in 726bff4.